presto-product-tests/conf/presto/etc/multinode-master-jvm.config

```diff
@@ -9,9 +9,8 @@
 # coordinator is not a worker
 -Xmx1G
 -XX:-UseBiasedLocking
--XX:+UseG1GC
--XX:G1HeapRegionSize=32M
--XX:+ExplicitGCInvokesConcurrent
+# Disable G1 to enable UseGCOverheadLimit (JDK-8212084) and reduce JVM's memory footprint
+-XX:-UseG1GC
```

Comment:
It is unclear what GC algorithm is used in this case, and it may change from one version of the VM to the other. Let's specify it explicitly. The default (effective GC algorithm with `-XX:-UseG1GC` ) for Java 8 is the parallel GC `-XX:+UseParallelGC`. Let's have it set explicitly.